src/style/CharityPage.less

```diff
@@ -194,9 +221,10 @@
 			display: flex;
 			justify-content: center;
 			align-items: center;
+			height: inherit;
```

Comment:
Is this fixing a glitch? To protect it from accidental edits in the future, please add a comment on what this line is for.